requirements/base.in

```diff
@@ -6,4 +6,3 @@ PyYAML
 redo
 scriptworker
 taskcluster
-towncrier
```

Comment:
Ah, removing `towncrier` makes sense. With the migration to CloudOps, we'll deploy a new version to staging with each master push and then tag for production. Only question I have is, what's the best way of keeping a CHANGELOG, going further, in the new world? Do we still want to?

Reply:
I didn't see `towncrier` used explicitly in CI. I assume it's used as a part of local development and you run it manually. In this case it doesn't belong to `base.in` (production) not `test.in` (CI). We can create `localdev.in` and put it there if needed. What do you think?

Reply:
We could do that. I guess my question is more generic, what's the best way going forward in the CloudOps world, to keep track of CHANGELOG? How do we solve this in Balrog?

Reply:
TBH, I'm not quiet sure if this is something related to the deployment approach. Maybe we need a check the current version is in listed in CHANGELOG?